README.md

```diff
@@ -43,6 +43,8 @@ Community contributed boards are:
 | [Kakute F7](http://www.holybro.com/product/kakute-f7-aio-v1-5/)                          | -           | [@amfern](https://github.com/amfern)           |         | `colcon.meta`            |
 | [STM32-E407](https://www.olimex.com/Products/ARM/ST/STM32-E407/resources/STM32-E407.pdf) | -           | [@dominikn](https://github.com/dominikn)       |         | `colcon.meta`            |
 
+You can [configure many parameters](https://micro.ros.org/docs/tutorials/advanced/microxrcedds_rmw_configuration/) of the `prebuild meta` by editing the respective file in the `extras/library_generation/` directory.
+
```

Comment:
Only if you rebuild the library. Please move this to the rebuild library section.